zebROS_ws/src/tf_object_detection/src/pub_video.py

```diff
@@ -24,7 +24,7 @@ def __init__(self):
             print("Pub topic is", self.pub_topic)
         if rospy.has_param('~filename'):
             self.filename = rospy.get_param('~filename')
-            print("Filename being used is", self.filename)
+            print("Filename being used is")
```

Comment:
Was this change intentional?  Seem like the actual file name would be useful info